Github/legos/github_get_issue/github_get_issue.py

```diff
@@ -0,0 +1,81 @@
+
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+
+import pprint
+from typing import Optional, List, Dict
+from pydantic import BaseModel, Field
+from github import GithubException
+
+
+
+class InputSchema(BaseModel):
+    owner: str = Field(
+        ..., description='Username of the GitHub user. Eg: "johnwick"', title='Owner'
+    )
+    repository: str = Field(
+        ...,
+        description='Name of the GitHub repository. Eg: "Awesome-CloudOps-Automation"',
+        title='Repository',
+    )
+    issue_number: int = Field(
+        ...,
+        description='Issue Number. Eg: "367"',
+        title='Issue Number',
+    )
+
+
+def github_get_issue_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+def github_get_issue(handle, owner:str, repository:str, issue_number:int) -> Dict:
+    """github_get_issue returns details of the issue
+
+        :type handle: object
+        :param handle: Object returned from task.validate(...).
+
+        :type owner: string
+        :param owner: Username of the GitHub user. Eg: "johnwick"
+
+        :type repository: string
+        :param repository: Name of the GitHub repository. Eg: "Awesome-CloudOps-Automation"
+
+        :type issue_number: int
+        :param issue_number: Issue number. Eg: 345
+
+        :rtype: Dict of issue details
+    """
+    issue_no = int(issue_number)
+    issue_details = {}
+    try:
+        owner = handle.get_user(owner)
+        repo_name = owner.login + '/' + repository
+        repo = handle.get_repo(repo_name)
+        issues = repo.get_issues()
+        for issue in issues:
```

Comment:
rather than getting all issues and filtering (we could run into issues on not in the top 100, cant we just call the issue specifically?
https://pygithub.readthedocs.io/en/latest/examples/Issue.html#get-issue

Reply:
Done